sys-testing/Cargo.toml

```diff
@@ -6,10 +6,13 @@ publish = false
 
```

Comment:
Should we rename this crate to `links-testing`?. Not really specific to `sys` anymore.